packages/react-conformance/tslint.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": ["@uifabric/tslint-rules"],
+  "rules": {}
```

For now you'll need this:
```suggestion
  "rules": {
    "interface-name": [true, "never-prefix"]
  }
```